user/languages/go.md

```diff
@@ -56,22 +69,25 @@ go_import_path: example.org/pkg/foo
 
 ## Dependency Management
 
-By default the install step defers to `go get ./...` or `go get -t ./...` if the version of go is greater than or equal
-to `1.2`.  If any of the following files are present, the default install step will be simply `true`:
+By default the install step defers to `go get ./...` or `go get -t ./...` if
```

Comment:
defers = defaults?

So, 

IF go >= 1.2 THEN go get -t ./..

IF go < 1.2 THEN go get ./..

IF makefile whatever THEN true

(and not make? )

Reply:
Yes, that's it:

(for make vs go get) https://github.com/travis-ci/travis-build/blob/125c5d1fa8ada1f0d79bb77bbccaefd1e7e43c13/lib/travis/build/script/go.rb#L86-L91
and

(forgo get vs go get -t) https://github.com/travis-ci/travis-build/blob/125c5d1fa8ada1f0d79bb77bbccaefd1e7e43c13/lib/travis/build/script/go.rb#L146-L152